java/ql/src/experimental/semmle/code/java/Logging.qll

```diff
@@ -28,6 +27,13 @@ class LoggingCall extends MethodAccess {
       m.getReturnType() instanceof VoidType and
       this = m.getAReference()
     )
+    or
+    exists(RefType t, Method m | t.hasQualifiedName("android.util", "Log") |
+      m.hasName(["d", "e", "i", "v", "w", "wtf"]) and
+      m.getDeclaringType() = t and
+      m.getReturnType() instanceof IntegralType and
```

Comment:
```suggestion
```
No need to constrain the return type at all, since all methods with these signatures are of interest.